### DISCUSSION TITLE
Not having to hold the ROG button to keep the program up

### DISCUSSION THREAD

**Discussion by bigggggert** [seerge/g-helper#3300] [Ideas]
It is kind of annoying that I have to hold the button to keep the screen up, I don't know if this is a bug or it is intended.

**Comment by seerge** [comment]
@bigggggert hello, what exactly do you mean? It's not clear.

**Reply by bigggggert** [reply]
Like when I press the ROG Key on my keyboard g-helper can stay up until I press it again and not until I top holding it, also "skip_aura" : 1, didn't work (for the Issues thread I made earlier)

**Reply by seerge** [reply]
@bigggggert app window shows when you click on ROG button. You don't need to hold it. 

As for your remark about `"skip_aura" : 1,` param it stops only aura init command on app start as I have explained. I advice you to reply about it in your original [issue](https://github.com/seerge/g-helper/issues/3299) thread.

**Reply by bigggggert** [reply]
> @bigggggert app window shows when you click on ROG button. You don't need to hold it.
> 
> As for your remark about `"skip_aura" : 1,` param it stops only aura init command on app start as I have explained.

Well I broke something then... Also I forgot to remove the comma on skip_aura sorry.

**Reply by seerge** [reply]
You don't need to remove comma. You need to maintain correct JSON structure when adding params. Check Power User settings [about how to do that](https://github.com/seerge/g-helper/wiki/Power-user-settings#general-note)

**Reply by bigggggert** [reply]
Ok the RGB problem is fixed, but I still dont know why it doesnt stay up when I press the ROG button.

**Reply by bigggggert** [reply]
And the RGB lighting change goes in steps of two instead of one.

**Reply by seerge** [reply]
@bigggggert check what happens inside `%appdata%\GHelper\log.txt` when you press your ROG button ? or upload full log here.

It could be that you have Asus Services running that kind of interfere by duplicating all keys (this would explain backlight changes in 2 steps). If so - stop them from `Extra` settings in the app.

**Reply by bigggggert** [reply]
It says 0 asus services running, and this is what happens when I press the ROG button:
10/25/2024 3:12:14 PM:  HDR: False 8 0 False
10/25/2024 3:12:14 PM: WMI event 56
10/25/2024 3:12:14 PM: Key: 56
And these are the RGB brightness keys:
10/25/2024 3:14:57 PM: Key: 197
10/25/2024 3:14:57 PM: WMI event 197
10/25/2024 3:14:57 PM: HotKey 1866: 5D-BA-C5-C4-02
10/25/2024 3:14:57 PM: HotKey 1866: 5D-BA-C5-C4-01
10/25/2024 3:14:57 PM: Key: 196
10/25/2024 3:14:57 PM: WMI event 196
10/25/2024 3:14:57 PM: HotKey 1866: 5D-BA-C5-C4-02
10/25/2024 3:14:57 PM: HotKey 1866: 5D-BA-C5-C4-03

**Reply by seerge** [reply]
@bigggggert ok, apparently your hardware duplicates hotkeys via both USB and WMI protocols. Some very old models are doing that.

Try this build
[GHelper.zip](https://github.com/user-attachments/files/17527196/GHelper.zip)

**Reply by bigggggert** [reply]
That works, thanks.

**Reply by bigggggert** [reply]
I have another problem, my GPU which is supposed to and was running at 1ghz is running at 300mhz and I cant make it run faster. The laptop is a GX501GI and the gpu is a 1080max-q.

**Reply by bigggggert** [reply]
10/25/2024 4:35:25 PM: ------------
10/25/2024 4:35:25 PM: App launched: Zephyrus GX501GI :0.197.0.0en-US.
10/25/2024 4:35:25 PM: Start Count: 10
10/25/2024 4:35:25 PM: Matrix control device was not found on your machine.
10/25/2024 4:35:26 PM: NVIDIA GeForce GTX 1080 with Max-Q Design
10/25/2024 4:35:27 PM: CPU: -1 - Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz - Intel64 Family 6 Model 158 Stepping 10
10/25/2024 4:35:27 PM: AutoSetting for Online
10/25/2024 4:35:27 PM: Eco flag : -65536
10/25/2024 4:35:27 PM: Mux flag : -65536
10/25/2024 4:35:27 PM: Input available: \\?\hid#vid_0b05&pid_1866&mi_02&col01#7&121e880&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030} 1866 64
10/25/2024 4:35:27 PM: \\?\DISPLAY#AUO71ED#5&2f9c6c78&0&UID4354#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7} DISPLAYCONFIG_OUTPUT_TECHNOLOGY_DISPLAYPORT_EMBEDDED
10/25/2024 4:35:27 PM: USB 1866|64: 5A-41-53-55-53-20-54-65-63-68-2E-49-6E-63-2E
10/25/2024 4:35:27 PM: \\?\DISPLAY#AUO71ED#5&2f9c6c78&0&UID4354#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7} \\.\DISPLAY1\Monitor0
10/25/2024 4:35:27 PM: Input: \\?\hid#vid_0b05&pid_1866&mi_02&col01#7&121e880&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
10/25/2024 4:35:27 PM:  HDR: False 8 0 False
10/25/2024 4:35:27 PM: Mode = 1 : 0
10/25/2024 4:35:27 PM: VivoMode = 2 : 0
10/25/2024 4:35:27 PM: Init 1866: 5D-B9
10/25/2024 4:35:27 PM: Init 1866: 5D-41-53-55-53-20-54-65-63-68-2E-49-6E-63-2E
10/25/2024 4:35:27 PM: Init 1866: 5D-05-20-31-00-1A
10/25/2024 4:35:27 PM:  HDR: False 8 0 False
10/25/2024 4:35:27 PM: Startup task scheduled: C:\Users\kuert\Desktop\GHelper.exe
10/25/2024 4:35:27 PM: Charge limit task scheduled: C:\Users\kuert\Desktop\GHelper.exe
10/25/2024 4:35:27 PM: GET GPU CLOCKS: 0, 0
10/25/2024 4:35:27 PM: GET CLOCK LIMIT: 0
10/25/2024 4:35:27 PM: Monitor Power On
10/25/2024 4:35:27 PM: Lid Open
10/25/2024 4:35:27 PM: Lid 1866: 5D-BA-C5-C4-03
10/25/2024 4:35:28 PM: Auto 1866: 5D-BA-C5-C4-03
10/25/2024 4:35:28 PM: Latest version 0.197.0.0
10/25/2024 4:35:33 PM: GET CLOCK LIMIT: 0
10/25/2024 4:35:33 PM: powershell nvidia-smi -lgc 0,2990
10/25/2024 4:35:34 PM: Setting locked GPU clocks is not supported for GPU 00000000:01:00.0. Treating as warning and moving on. All done.
10/25/2024 4:35:34 PM: GET GPU CLOCKS: 0, 0
10/25/2024 4:35:34 PM: GET GPU CLOCKS: 0, 0
10/25/2024 4:35:34 PM: GET CLOCK LIMIT: 0

**Reply by seerge** [reply]
@bigggggert i see that you have manually set a GPU clock limit under `Fans+Power`, you either need to click `Factory Defaults` or drag slider back to it's default (OFF) position

```
10/25/2024 4:35:33 PM: powershell nvidia-smi -lgc 0,2990 <--- LIMIT
```

**Reply by bigggggert** [reply]
> @bigggggert i see that you have manually set a GPU clock limit under `Fans+Power`, you either need to click `Factory Defaults` or drag slider back to it's default (OFF) position
> 
> ```
> 10/25/2024 4:35:33 PM: powershell nvidia-smi -lgc 0,2990 <--- LIMIT
> ```

![image](https://github.com/user-attachments/assets/92919341-6f57-4ec6-8f8d-7d8751fc6b80)
its already at the default, already tried resseting to default and it doesnt do anything because its already at default

**Reply by seerge** [reply]
@bigggggert ok, probably you dragged slider before ? 

Anyway, it looks like clock limiting is not even supported on your device (as nvidia-smi tool that sets it says so)
```
10/25/2024 4:35:34 PM: Setting locked GPU clocks is not supported for GPU 00000000:01:00.0. Treating as warning and moving on. All done.
```

Besides that there is nothing that you could set in G-Helper that would limit clocks :) Keep in mind that when not in use GPU most likely won't run on max clocks

**Reply by bigggggert** [reply]
Yea but It was 300 mhz when running furmark, I'm gonna go see if any of the other programs I have are causing this.

**Reply by seerge** [reply]
Try to reboot, may be it's just some driver hiccup.

**Reply by bigggggert** [reply]
Yep that fixed it, was hoping to overclock using G-Helper but the gpu is throttling so I cant ;c, wish my laptop let me change fan curve.
